src/WebJobs.Extensions.ApiHub/ApiHubFileAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.ApiHub.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
```

Comment:
Move to the root Microsoft.Azure.WebJobs namespace for discoverability. See all the other extension attributes (e.g. SendGrid) for example.